src/app/components/button/button.tsx

```diff
@@ -1,13 +1,23 @@
+import { Arrow } from '@/icons/arrow';
+
 import './button.scss';
 
-type ButtonProps = {
+export type ButtonProps = {
   label: string;
   href: string;
 };
 
-export const Button = ({ label, href }: ButtonProps) => (
-  <a className="button" href={href} target="_blank" rel="noreferrer">
-    <span className="label">{label}</span>
-    <span className="material-symbols-outlined arrow">arrow_forward</span>
-  </a>
-);
+export const MakeBtnTemp = (isOutlined: boolean) => {
+  const btnClass = isOutlined ? 'button outlined' : 'button colored';
+  return ({ label, href }: ButtonProps) => (
+    <a className={btnClass} href={href} target="_blank" rel="noreferrer">
+      {isOutlined ? label : <span className="label">{label}</span>}
+      <span className="arrow">
+        <Arrow />
+      </span>
+    </a>
+  );
+};
+
+export const Button = MakeBtnTemp(false);
+export const ButtonOutlined = MakeBtnTemp(true);
```

Comment:
How about rewriting it to something like this:
```
export type ButtonProps = {
  label: string;
  href?: string;
  outlined?: boolean;
  size?: 'small' | 'medium' | 'large';
  icon?: React.ReactNode;
};

export const Button = (props: ButtonProps) => {
  const {
    label,
    href,
    outlined = false,
    size = 'medium',
    icon = (
      <span className="arrow">
        <Arrow />
      </span>
    ),
  } = props;

  const classes = classNames('button', {
    outlined,
    colored: !outlined,
    small: size === 'small',
    medium: size === 'medium',
    large: size === 'large',
  });

  return (
    <a className={classes} href={href} target="_blank" rel="noreferrer">
      <span className="label">{label}</span>
      {icon}
    </a>
  );
};
```

And then you can reuse it for the following buttons using `size='small'`:
![image](https://github.com/rolling-scopes/site/assets/69504277/d4d73a09-d57a-4f8f-821c-920c7c8a941a)

Reply:
Yes, thank you.

Reply:
Oh, I suggest it above too)))) I deleted my comment)) I used `classnames` too.